soc/arm/nuvoton_npcx/common/pinctrl_soc.h

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2022 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_PINCTRL_SOC_H_
+#define _NUVOTON_PINCTRL_SOC_H_
+
+#include <devicetree.h>
+#include <sys/util_macro.h>
+#include <zephyr/types.h>
```

Comment:
nit: please update to make use of `<zephyr/`> prefix

Reply:
Done.